linkerd/app/inbound/src/detect.rs

```diff
@@ -22,6 +23,15 @@ pub struct Tls {
     permit: Permitted,
 }
 
+#[derive(Clone, Debug)]
+struct Detect {
+    timeout: time::Duration,
+    tls: Tls,
+}
+
+#[derive(Copy, Clone, Debug)]
+struct ConfigureHttpDetect;
```

Comment:
guessing that subsequent changes will make this actually do something?

Reply:
It's used now -- this is the ExtractParam that configures the http detect stack